### PR TITLE
PMM-7495 Release Notes 2.15.0

### PR DESCRIPTION
**Pull request by PaulJacobs-percona**

### CODE REVIEW

File: docs/release-notes/2.15.0.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+* [PMM-7331](https://jira.percona.com/browse/PMM-7331): Security Threat Tool: Failed checks have 'Read more' links with helpful content
+* [PMM-7329](https://jira.percona.com/browse/PMM-7329): Security Threat Tool: Hide check results tab if STT is disabled
+* [PMM-7328](https://jira.percona.com/browse/PMM-7328): Security Threat Tool: Download and run checks when activated, immediately, repeating every 24 hours thereafter (Previously, downloading and running new checks happened every 24 hours but the cycle didn't begin when STT was activated.)
+* [PMM-7259](https://jira.percona.com/browse/PMM-7259): Integrated Alerting: 'Alerting is disabled' message gives hint to enable it
+* [PMM-7257](https://jira.percona.com/browse/PMM-7257): Integrated Alerting: Easier-to-read rule details in Alert Rules list (API to expose Alert Rule's expression and real values)
+* [PMM-7240](https://jira.percona.com/browse/PMM-7240): DBaaS: Change from Dashboard to Grafana Page -- We changed the DBaaS page from a Grafana Dashboard to a Grafana Page to be better aligned with the DBaaS enable/disable status and avoid confusion when DBaaS is disabled.
```
Comment (shoffman-percona) [inline docs/release-notes/2.15.0.md:61]: I'm concerned we're blurring the lines between features that are GA and what's still very much a tech preview...maybe nothing to do but these are still not GA Features so does mixing them in with GA stuff send the wrong impression?

File: docs/release-notes/2.15.0.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+* [PMM-6711](https://jira.percona.com/browse/PMM-6711): Add filter for `external_group` in `pmm-managed` services list (API)
+* [PMM-6695](https://jira.percona.com/browse/PMM-6695): Update Grafana 7.1.3 to 7.3.7 (See [What's new in Grafana 7.2](https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v7-2) and [What's new in Grafana 7.3](https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v7-3/).)
+* [PMM-5888](https://jira.percona.com/browse/PMM-5888): Improve `pmm-admin --help` descriptions for external services
```
Comment (shoffman-percona) [inline docs/release-notes/2.15.0.md:64]: How is this materially different from PMM-7476?

**Review by shoffman-percona — CHANGES_REQUESTED** [review]

File: docs/release-notes/2.15.0.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## New Features
+
+* [PMM-7412](https://jira.percona.com/browse/PMM-7412): DBaaS: Ability to see DB Cluster creation/modification logs (API) (Thanks to Jan Prukner for reporting this issue)
```
[the remark this answers is not in the document]
Comment (PaulJacobs-percona) [reply docs/release-notes/2.15.0.md:25]: Jirabot makes this mistake in a few places. I will manually remove them.

File: docs/release-notes/2.15.0.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+* [PMM-6797](https://jira.percona.com/browse/PMM-6797): Ability to add HAProxy services with `pmm-admin [inventory] add [service] haproxy` command
+* [PMM-7496](https://jira.percona.com/browse/PMM-7496): Dashboards: New HAProxy PXC dashboards
+* [PMM-7418](https://jira.percona.com/browse/PMM-7418): Security Threat Tool: Ability to disable *all* security checks
+* [PMM-7417](https://jira.percona.com/browse/PMM-7417): Security Threat Tool: Show list of all available security checks
+* [PMM-7405](https://jira.percona.com/browse/PMM-7405): Check connection response format when adding external agents
+* [PMM-7380](https://jira.percona.com/browse/PMM-7380): Backup: Locations: database model, List locations, Add locations (API) (**Note:** This API is subject to change. It exists in development versions and will not be exposed in future stable versions.)
+* [PMM-7296](https://jira.percona.com/browse/PMM-7296): Integrated Alerting: Pagination device for viewing large numbers of Alert Rules (API)
+* [PMM-7294](https://jira.percona.com/browse/PMM-7294): Integrated Alerting: Pagination device for viewing large numbers of Alert Rules (UI)
```
[the remark this answers is not in the document]
Comment (PaulJacobs-percona) [reply docs/release-notes/2.15.0.md:34]: @rnovikovP has an outstanding task to merge UI/API/FE/BE dups.

File: docs/release-notes/2.15.0.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+* [PMM-7181](https://jira.percona.com/browse/PMM-7181): Dashboards: MySQL/MongoDB/PostgreSQL Instance Summary: Add panel with `pt-*-summary` information
```
[the remark this answers is not in the document]
Comment (PaulJacobs-percona) [reply docs/release-notes/2.15.0.md:35]: Good point. And I might even list them all explicitly.

File: docs/release-notes/2.15.0.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+* [PMM-7177](https://jira.percona.com/browse/PMM-7177): Dashboards: New Grafana panel showing all `pt-*-summary` outputs in one place
+* [PMM-7123](https://jira.percona.com/browse/PMM-7123): Add `external-serverless` instances (UI)
+* [PMM-6943](https://jira.percona.com/browse/PMM-6943): Show HAProxy type services in PMM Inventory (UI)
+* [PMM-6924](https://jira.percona.com/browse/PMM-6924): Show page 'breadcrumbs' navigation aid on non-dashboard pages as well as Grafana dashboard pages
+* [PMM-6671](https://jira.percona.com/browse/PMM-6671): Ability to add HAProxy instances (UI)
+* [PMM-6640](https://jira.percona.com/browse/PMM-6640): Ability to add HAProxy services (API) (Technical Preview)
```
[the remark this answers is not in the document]
Comment (PaulJacobs-percona) [reply docs/release-notes/2.15.0.md:41]: Turns out to be an error that will be cured in the next jirabot run.

File: docs/release-notes/2.15.0.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+* [PMM-5778](https://jira.percona.com/browse/PMM-5778): Check connection response format when adding external agents (UI)
```
[the remark this answers is not in the document]
Comment (PaulJacobs-percona) [reply docs/release-notes/2.15.0.md:42]: One of Roma's deliberate dups that helps get ticket titles cleaned up.

File: docs/release-notes/2.15.0.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+* [PMM-4172](https://jira.percona.com/browse/PMM-4172): Ability to execute `pt-mysql-summary` for MySQL service (API)
+
+
+
+## Improvements
+
+* [PMM-6693](https://jira.percona.com/browse/PMM-6693): pmm-managed denies request when using API keys even if the role set is admin -- The PMM API will now accept requests not only with username/password but also with API tokens generated in Grafana. 
+* [PMM-7487](https://jira.percona.com/browse/PMM-7487): Check connection to HAProxy services when adding them for monitoring (UI) (Thanks to Jiří Čtvrtka for reporting this issue)
```
[the remark this answers is not in the document]
Comment (PaulJacobs-percona) [reply docs/release-notes/2.15.0.md:50]: I'll remove when the last extract is done.

File: docs/release-notes/2.15.0.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+## New Features
+
+* [PMM-7412](https://jira.percona.com/browse/PMM-7412): DBaaS: Ability to see DB Cluster creation/modification logs (API) (Thanks to Jan Prukner for reporting this issue)
+* [PMM-7419](https://jira.percona.com/browse/PMM-7419): DBaaS: Ability to see DB Cluster creation/modification logs (UI)
+* [PMM-6797](https://jira.percona.com/browse/PMM-6797): Ability to add HAProxy services with `pmm-admin [inventory] add [service] haproxy` command
+* [PMM-7496](https://jira.percona.com/browse/PMM-7496): Dashboards: New HAProxy PXC dashboards
+* [PMM-7418](https://jira.percona.com/browse/PMM-7418): Security Threat Tool: Ability to disable *all* security checks
+* [PMM-7417](https://jira.percona.com/browse/PMM-7417): Security Threat Tool: Show list of all available security checks
+* [PMM-7405](https://jira.percona.com/browse/PMM-7405): Check connection response format when adding external agents
```
[the remark this answers is not in the document]
Comment (PaulJacobs-percona) [reply docs/release-notes/2.15.0.md:31]: Perhaps clearer by deleting 'format'. It's the connection check that's important. But the user story is a little thin:
>  As a user I want to be able to check connection to external exporter

File: docs/release-notes/2.15.0.md [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+* [PMM-7380](https://jira.percona.com/browse/PMM-7380): Backup: Locations: database model, List locations, Add locations (API) (**Note:** This API is subject to change. It exists in development versions and will not be exposed in future stable versions.)
```
[the remark this answers is not in the document]
Comment (rnovikovP) [reply docs/release-notes/2.15.0.md:32]: Added  SKIP. 
Until we'll have MVP nothing to tell

File: docs/release-notes/2.15.0.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (rnovikovP) [reply docs/release-notes/2.15.0.md:34]: I think the final version should  have them combined

File: docs/release-notes/2.15.0.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (rnovikovP) [reply docs/release-notes/2.15.0.md]: this is API . corrected the title

File: docs/release-notes/2.15.0.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (rnovikovP) [reply docs/release-notes/2.15.0.md:43]: let's do resort and de-dup at the last stage

File: docs/release-notes/2.15.0.md (hunk 1 above)
Comment (rnovikovP) [reply docs/release-notes/2.15.0.md:63]: This is for the last iteration when we'll do re-ordering.
 we need to group issue  by features  (released ones) then all Tech preview related

File: docs/release-notes/2.15.0.md (hunk 2 above)
Comment (rnovikovP) [reply docs/release-notes/2.15.0.md:66]: removed  https://jira.percona.com/browse/PMM-7476 
Everything  will be in one  issue

File: docs/release-notes/2.15.0.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (PaulJacobs-percona) [reply docs/release-notes/2.15.0.md]: @rnovikovP changed the title back to "Check connection response format when adding External Service to monitoring (API)" from "Check connection when adding external agents".
@shoffman-percona Which do you prefer?

File: docs/release-notes/2.15.0.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+* [PMM-6681](https://jira.percona.com/browse/PMM-6681): Not all PMM admin users can download diagnostic logs, only those with Grafana admin rights
+* [PMM-6685](https://jira.percona.com/browse/PMM-6685): In low screen resolutions Services submenu wraps, becomes obscured, and can't be accessed
+* [PMM-7227](https://jira.percona.com/browse/PMM-7227): Table stats metrics not being collected in instances with millions of tables
+* [PMM-7426](https://jira.percona.com/browse/PMM-7426): `vmagent` continually restarts, blocking comms between `pmm-agent` & `pmm-managed` -- Users running multiple services on the same PMM agent in 'push' mode could face this issue when restarting the agent after bulk-adding services.
+* [PMM-7470](https://jira.percona.com/browse/PMM-7470): MongoDB exporter `IndexStatsCollections` is assigned values from wrong flag (Thanks to Tim for reporting this issue)
+* [PMM-7119](https://jira.percona.com/browse/PMM-7119): Integrated Alerting: Update error notification for adding/update Alert rule Template -- There was inconsistent behavior if you tried to add a new Rule Template with an already-used name.
```
Comment (rnovikovP) [inline docs/release-notes/2.15.0.md:76]:
```suggestion
* [PMM-7119](https://jira.percona.com/browse/PMM-7119): Integrated Alerting: Update error notification for adding/update Alert rule Template -- There was inconsistent behavior if you tried to add a new Rule Template with an already-used name.
```

File: docs/release-notes/2.15.0.md [hunk 12]
```diff
@@ -0,0 +1,120 @@
...
+
+	_Disable collectors during adding node/service to monitoring_
+
+	With this feature users can disable any collector used by PMM to get metrics. When metrics cannot be collected or are no longer needed, disabling the collector(s) prevents PMM from flooding logs and saves infrastructure resources.
+
+	Our vision for PMM collectors is to provide “stop from collecting” functionality to prevent possible harm to the user environment. This “disable” feature is an initial step towards the ideal functionality. The full and flexible management for “What metrics to collect and in what resolution” is slated for future releases.
+
+	_External services monitoring_
+
+	Since PMM v1.4.0, users had the ability to monitor external services Percona didn’t currently support (e.g., Redis). This blog article from 2018 nicely described external services monitoring at that time. (At that time Percona was not natively supporting a PostgreSQL monitoring service and so this was listed as an external service. Today, PostgreSQL is natively supported by PMM.)
```
Comment (PaulJacobs-percona) [inline docs/release-notes/2.15.0.md:35]: @rnovikovP What's the link for the blog post mentioned here?

Comment (rnovikovP) [reply docs/release-notes/2.15.0.md:35]: https://www.percona.com/blog/2018/02/12/percona-monitoring-management-pmm-support-external-monitoring-services-yes/

**Pull request merged**